cmd/heartbeat/heartbeat.go

```diff
@@ -151,6 +151,7 @@ func buildHeartbeats(params paramscmd.Params) []heartbeat.Heartbeat {
 	}
 
 	heartbeats = append(heartbeats, heartbeat.New(
+		params.Heartbeat.Project.BranchAlternate,
```

Comment:
Are these params alphabetized? If so, it should go just after `Project.Alternate`.

Reply:
oops!